linux_os/guide/services/sssd/sssd_enable_pam_services/ansible/shared.yml

```diff
@@ -0,0 +1,41 @@
+# platform = multi_platform_all
+# reboot = false
+# strategy = configure
+# complexity = low
+# disruption = medium
+
+- name: Find all the conf files inside the /etc/sssd/conf.d/ directory
+  ansible.builtin.find:
+    paths:
+    - "/etc/sssd/conf.d/"
+    patterns: "*.conf"
+  register: sssd_conf_d_files
+
+- name: Modify lines in files in the /etc/sssd/conf.d/ directory
+  ansible.builtin.replace:
+    path: "{{ item }}"
+    regexp: '^(services\s*=.*)'
+    replace: '\1,pam'
```

Comment:
Since the group in `regexp` is collecting the whole line, appending `pam` to this referenced group should duplicate the `pam` value every time the Playbook is executed. Have you tested this?